samples/04_gis_analysts_data_scientists/building_damage_assessment_using_feature_classifier.ipynb

```diff
@@ -60,7 +60,12 @@
    "metadata": {},
```

Comment:
<p><code>input_raster=input_raster_layer</code>  should work... do we need to use <code>input_raster_layer.url</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>